platforms/core-runtime/logging/src/testFixtures/groovy/org/gradle/internal/logging/ToStringLogger.groovy

```diff
@@ -35,16 +36,46 @@ class ToStringLogger extends OutputEventListenerBackedLogger {
         log(message)
     }
 
+    @Override
+    void lifecycle(String format, Object... args) {
```

Comment:
💭 All these methods previously no-opped...now they properly log to the StringBuilder, which is useful for testing.